public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
Nora I'm confused by your question about forms disappearing.. Can you elaborate on that a bit?